llama_stack/providers/utils/inference/openai_mixin.py

```diff
@@ -111,6 +111,18 @@ def get_extra_client_params(self) -> dict[str, Any]:
         """
         return {}
 
+    async def get_models(self) -> Iterable[str] | None:
```

Comment:
can we make this name a bit easier to understand w.r.t. the rest of the methods we have? `get_` vs `list_` etc? it is not very clear what the semantics of this one is immediately.